README.md

```diff
@@ -354,20 +359,75 @@ Submit the case (see CollectionCase.java) by swapping `<EVENT_JSON>` from above
 
 And send And send `python publisher.py local publish event_case-update`
 
+4) **Survey Update**
+
+Submit the case (see SurveyUpdate.java) by swapping `<EVENT_JSON>` from above to the json below (including the tripple quotes)
+
+    """{
+        "event" : {
+          "type" : "SURVEY_UPDATE",
+          "source" : "SAMPLE_LOADER",
+          "channel" : "RH",
+          "dateTime" : "2020-06-08T07:28:45.117Z",
+          "transactionId" : "c45de4dc-3c3b-11e9-b210-d663bd873d93"
+        },
+        "payload": {
+          "surveyUpdate": {
+            "surveyId": "3883af91-0052-4497-9805-3238544fcf8a",
+            "name": "Excepteur veniam"
+          }
+        }
+      }
+    """
+
+And send And send `python publisher.py local publish event_survey-update`
+
+5) **Collection Exercise Update**
+
+Submit the case (see SurveyUpdate.java) by swapping `<EVENT_JSON>` from above to the json below (including the tripple quotes)
+
+    """{
```

Comment:
doesn't render right.  Probably need triple single quotes instead of double quotes

Reply:
I've rewritten this section to make it shorter and hopefully easier to follow